src/llmcompressor/transformers/compression/helpers.py

```diff
@@ -111,36 +104,6 @@ def infer_sparsity_structure_from_model(model: torch.nn.Module) -> Optional[str]
     return None
 
 
-def hessian_memory_requirements(model: torch.nn.Module) -> int:
-    """
-    Determines the number of bytes needed to store Hessian data for a single
-    transformer layer in model. This is used for reserving memory for GPTQ
-    quantization
-
-    :param model: model to calculate requirements for
-    :return: number of bytes required to reserve for GPTQ on a single layer
-    """
-    transformer_layers = get_layers(get_no_split_params(model), model)
-    total_hessian_elems = {}
-    max_column_size = {}
-    for no_split_name, no_split_layer in transformer_layers.items():
-        total_hessian_elems[no_split_name] = 0
-        max_column_size[no_split_name] = 0
-        for _name, module in no_split_layer.named_modules():
-            if isinstance(module, Linear) and hasattr(module, "weight"):
-                column_size = module.weight.shape[1]
-                total_hessian_elems[no_split_name] += column_size * column_size
-                if column_size > max_column_size[no_split_name]:
-                    # max extra memory for inverse calculation
-                    max_column_size[no_split_name] = column_size
-
-    max_total_hessian_elems = max(total_hessian_elems.values())
-    overall_max_column_size = max(max_column_size.values())
-    bytes_per_weight = 32 // 8  # hessians are float32
-    inverse_reserved = overall_max_column_size * overall_max_column_size
-    return (max_total_hessian_elems + inverse_reserved) * bytes_per_weight
-
-
 def quantization_memory_requirement(model: torch.nn.Module) -> int:
```

Comment:
do we still need quantization_memory_requirements?

Reply:
No, thanks for the catch